test/rebar_dir_SUITE.erl

```diff
@@ -15,8 +16,19 @@
 all() -> [default_src_dirs, default_extra_src_dirs, default_all_src_dirs,
           src_dirs, extra_src_dirs, all_src_dirs,
           profile_src_dirs, profile_extra_src_dirs, profile_all_src_dirs,
-          retarget_path, alt_base_dir_abs, alt_base_dir_rel].
-
+          retarget_path, alt_base_dir_abs, alt_base_dir_rel, global_cache_dir,
+          default_global_cache_dir, overwrite_default_global_cache_dir].
+
+init_per_testcase(default_global_cache_dir, Config) ->
+    [{apps, AppsDir}, {checkouts, CheckoutsDir}, {state, _State} | Config] = rebar_test_utils:init_rebar_state(Config),
+    NewState = rebar_state:new([{base_dir, filename:join([AppsDir, "_build"])}
+                            ,{root_dir, AppsDir}]),
+    [{apps, AppsDir}, {checkouts, CheckoutsDir}, {state, NewState} | Config];
+init_per_testcase(overwrite_default_global_cache_dir, Config) ->
+    [{apps, AppsDir}, {checkouts, CheckoutsDir}, {state, _State} | Config] = rebar_test_utils:init_rebar_state(Config),
+    NewState = rebar_state:new([{base_dir, filename:join([AppsDir, "_build"])}
+                            ,{root_dir, AppsDir}]),
+    [{apps, AppsDir}, {checkouts, CheckoutsDir}, {state, NewState} | Config];
```

Comment:
I was wondering if it's better to have these two function duplicated or merge everything in one using a case like this:

```erlang
init_per_testcase(TestCase, Config) ->
    case lists:member(TestCase, [default_global_cache_dir, overwrite_default_global_cache_dir]) of
...
```